application/models/TokenDynamic.php

```diff
@@ -796,6 +797,19 @@ public function getAttributesForGrid()
         return array_merge($this->standardColsForGrid, $aCustomAttributesCols);
     }
 
+    public static function formatAttribute($data, $name)
```

Comment:
Doc?

Reply:
> Maybe adding it in the LSActiveRecord as a non-static function?

:+1: for LSActiveRecord, unsure about non-static , I

>  At last, the length could be a paameters as well ?

Don't know, since it's more related to display, css part fix the length show too. But I can add a config for that.

Reply:
About name default is https://www.yiiframework.com/doc/api/1.1/CFormatter#formatText-detail then : formatLongText seems OK.

Reply:
Move to formatter https://www.yiiframework.com/doc/api/1.1/CGridView#formatter-detail

Reply:
>  At last, the length could be a paameters as well ?

Can be set in config, but don't found the way to set formatter params easily in [GridView](https://www.yiiframework.com/doc/api/1.1/CGridView) ?